### PR TITLE
Improve snapshot release instructions

### PR DESCRIPTION
**Pull request by dskloetd**
# Motivation

The instructions in README.md to release a new dfx snapshot required modifying a command before executing it.

# Changes

Provide instructions that can be copied and used as-is.

### CODE REVIEW

**Comment by bitdivine** [comment]
The aggregator release instructions are on Notion.  These here are not used and likely to mislead.  They should really be deleted, so that there is one source of truth.

File: README.md [hunk 1]
```diff
@@ -40,8 +40,10 @@ the parameters passed to `ic-admin` from `bin/dfx-sns-sale-propose`.
 A new stock snapshot can be released by pushing a tag:
 
 ```
-git tag -a release-2013-45-56 -m "The new release"
-git push origin release-2013-45-56
+RELEASE="release-$(date +"%Y-%m-%d")"
+echo "$RELEASE"
+git tag -a "$RELEASE" -m "Release a new dfx snapshot"
+git push origin "$RELEASE"
```
[the remark this answers is not in the document]
Comment (dskloetd) [reply README.md:46]: I don't know. This is how I've been making snapshots so far. This PR doesn't change anything in that respect, does it? What would you suggest to do instead?

**Comment by dskloetd** [comment]
>  These here are not used and likely to mislead.

What do you mean by "not used"? I use them.

File: README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (bitdivine) [reply README.md:46]: Git tag without annotation.  Yes, this should have been caught sooner but better now than never.

**Comment by bitdivine** [comment]
Regarding different instructions, there should be one source of truth for how to deploy.  That one source of truth is in notion.  If you like you can put a link from here to notion.

**Review by bitdivine — APPROVED** [review]

**Pull request merged**